bricks/presentational-bricks/src/markdown-editor/MarkdownEditor.tsx

```diff
@@ -65,10 +78,14 @@ export function MarkdownEditorItem(
           : placeholderText;
         setValue(newValue);
         triggerChange(newValue);
+        const { width, height } = await getImageInfo(file);
+        const useMinSize = width >= 1024 && height >= 768;
         // 上传文件
         try {
           const response = await ObjectStoreApi_putObject(props.bucketName, {
             file: file,
+            width: useMinSize ? 1024 : width / 2,
+            height: useMinSize ? 768 : height / 2,
```

Comment:
这里前端不用管这里图片的尺寸，只需要给后台一个 `width/height` 的最大界限值就好了吧？

另外 1024/768 的限制有点小了，建议这样：
```suggestion
            width: 1280,
            height: 800,
```